packages/postcss-minify-font-values/src/lib/minify-family.js

```diff
@@ -33,9 +33,9 @@ const regexSimpleEscapeCharacters = /[ !"#$%&'()*+,.\/;<=>?@\[\\\]^`{|}~]/;
 
 function escape(string, escapeForString) {
   let counter = 0;
-  let character = null;
-  let charCode = null;
-  let value = null;
+  let character;
```

Comment:
There's no checks against `null` specifically so assigning to `null` just adds one possible value to the type for no gain.